tests/readcsr.rs

```diff
@@ -155,7 +155,7 @@ fn read_csr_with_custom_extension() {
                 assert_eq!(req.extensions.len(), 1);
                 let extension = req.extensions.first().unwrap();
                 assert_eq!(extension.oid, OID_CUSTOM_EXTENSION);
-                assert_eq!(extension.critical, false);
+                assert!(!extension.critical);
```

Comment:
I think the two `assert_eq!` fixes in this file are sensible, but it'd be nicer for review purposes if it was separated into its own commit since it's unrelated to the lifetime elision changes.

Reply:
Haha, my opinion was to think of this as minor, but it's true it doesn't match commit title. I'll split the patch